### ISSUE TITLE
Multiple ListBindingSectionController updates in quick succession makes completion block fail

### ISSUE THREAD

**Issue by yusuftor** [Instagram/IGListKit#1234]
I have real time updates to the data within my `ListBindingSectionController`. This involves retrieving data from a server, updating local variables, then calling `self.update(animated: false)`.

When the real-time updates from the server is retrieved in quick succession, sometimes the `update(animated:)` function is not completed. This leads to inconsistencies within the section controller. Is there any way to make sure that all updates will be completed, like queueing them?

**Comment by rnystrom** [comment]
What do you mean not complete? Like the completion block?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by yusuftor** [comment]
Sorry, I mean that the completion block gets called but the `completed` variable returned from the update function is `false`. So I guess the update function gets cancelled because there’s already an update going on.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by guluoguawenWjy** [comment]
> Sorry, I mean that the completion block gets called but the `completed` variable returned from the update function is `false`. So I guess the update function gets cancelled because there’s already an update going on.
> 
> Sent with [GitHawk](http://githawk.com)

ListBindingSectionController Not have Queue, I have the same problem。

**Comment by rnystrom** [comment]
Yup, this looks like a bug. We should be queuing `completion` blocks and executing all of them on unload.
